vue-frontend/src/nolonely-data.js

```diff
@@ -1,7 +1,7 @@
 import landingBackground400w from "@/assets/images/portfolio/nolonely/landing/background-400w.webp";
```

Comment:
I think you have deleted 400 and 800 width images. Please check all images once

Reply:
All the files are there.I have checked thoroughly.